### ISSUE TITLE
org.dslforge.workspace.config.configuration 

### ISSUE THREAD

**Issue by mspiffy** [plugbee/dslforge#45]
I modified/then deleted my org.dslforge.workspace.config.configuration extension point (and path contribution). Recreating an org.dslforge.workspace.config.configuration - where can I find the schema for the extension point; without a schema I get a java.sql.SQLException in terms of the workspace going to the default value.  Which schema file could one apply to this config extension point?

**Comment by jabiercoding** [comment]
The schema is here: https://github.com/plugbee/dslforge/blob/master/plugins/org.dslforge.workspace.config/schema/configuration.exsd
but I think that you just want to add the extension point again, right?
In the web plugin you can add
```
<extension
        point="org.dslforge.workspace.config.configuration">
     <contribution
           path="C:\www\somefolder">
     </contribution>
  </extension>
```


for the path, there are these examples in the extension point description
D:/www/dslforge/workspace or /var/lib/tomcat7/workspace

**Comment by kgardas** [comment]
Slightly related, is it possible to modify path programatically? Or is it possible to use @user.home to point to the user directory?

**Comment by jabiercoding** [comment]
I agree. If we are sharing the project in a team during development or if we have it in several computers we should not rely on hardcoded absolute paths. Right now, if we don't define this extension point it will take `D:/www/dslforge/workbench` as default so we need to have Windows and a D disk. See https://github.com/plugbee/dslforge/blob/0888041d71e2550795a6c271f612953902b79f22/plugins/org.dslforge.workspace/src/org/dslforge/workspace/IWorkspaceConstants.java#L28 
As @kgardas suggests, at least we can replace this line with
`public static final String WORKSPACE_DEFAULT_PATH = System.getProperty("user.home") + File.separator + "www" + File.separator + "dslforge" + File.separator + "workbench";`
@kgardas, is this solution valid for your case? If yes, please, open a new issue to manage it correction properly or to continue this discussion.
